tools/cldr-code/src/main/java/org/unicode/cldr/util/ChainedMap.java

```diff
@@ -103,9 +129,18 @@ public Iterator<Entry<K3, Map<K2, Map<K1, V>>>> iterator() {
         public Iterable<Row.R4<K3, K2, K1, V>> rows() {
             List<R4<K3, K2, K1, V>> result = new ArrayList<>();
             for (Entry<Object, Object> entry0 : super.mapBase.entrySet()) {
-                for (Entry<Object, Object> entry1 : ((Map<Object, Object>) entry0.getValue()).entrySet()) {
-                    for (Entry<Object, Object> entry2 : ((Map<Object, Object>) entry1.getValue()).entrySet()) {
-                        R4<K3, K2, K1, V> item = (R4<K3, K2, K1, V>) Row.of(entry0.getKey(), entry1.getKey(), entry2.getKey(), entry2.getValue());
+                for (Entry<Object, Object> entry1 : (
+                    (Map<Object, Object>) entry0.getValue()
+                ).entrySet()) {
```

Comment:
Why on earth is it breaking (...).method() before ")."?

Reply:
looks like it's balancing parentheses